### PR TITLE
added some Styling

### PR DESCRIPTION
**Pull request by KOlofinmoyin**

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,38 @@
  *= require_tree .
  *= require_self
  */
+
+ body {
+ font-family: 'Tenor Sans', sans-serif;
+ background-color:  	#f7f7f7;
+ }
+
+ .p1{border-width:10px;border-style:solid;padding:5px;border-color:green;}
```
Comment (emmavanoss) [inline app/assets/stylesheets/application.css]: Can we insert some line breaks please?

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,38 @@
...
+
+ /* Add a green background colour for nav background */
```
Comment (emmavanoss) [inline app/assets/stylesheets/application.css]: Very handy comments!

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,3 +1,12 @@
+<link href="https://fonts.googleapis.com/css?family=Tenor+Sans&display=swap" rel="stylesheet">
+<link rel="stylesheet" type="text/css" href="base.css">
+
+<div class="topnav">
+  <a class="active" href="#home">Home</a>
+  <a href="#create">Create</a>
+<a href="#friends">Friends</a>
```
Comment (emmavanoss) [inline app/views/posts/index.html.erb]: We'll want to update these when we add more routes but I think they're OK as placeholders for now

File: app/views/posts/new.html.erb [hunk 4]
```diff
@@ -1,3 +1,12 @@
+<link href="https://fonts.googleapis.com/css?family=Tenor+Sans&display=swap" rel="stylesheet">
+<link rel="stylesheet" type="text/css" href="base.css">
+
+<div class="topnav">
+  <a class="active" href="#home">Home</a>
+  <a href="#create">Create</a>
+<a href="#friends">Friends</a>
+</div>
+
```
Comment (emmavanoss) [inline app/views/posts/new.html.erb]: We could consider putting these in a template for all pages. Elfi might have an idea how to do this

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (robertwoolley99) [reply app/assets/stylesheets/application.css]: Artefact left over from testing out borders. Removed.

**Review by emmavanoss — APPROVED** [review]

**Pull request merged**